lighting.py

```diff
@@ -103,6 +105,9 @@ def attributeChangeCallback(
     size: int,
     value: bytes,
 ):
+    if not value:
```

Comment:
It will be hard to understand such code because it doesn't belong here. Such conditional work needs to happen in the right scope, before calling the device function or inside them. Defer to https://github.com/farshidtz/matter-bridge-tapo/issues/4